graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/migration/CreateUdtQuery.java

```diff
@@ -51,6 +57,13 @@ public String getDescription() {
     return "Create UDT " + type.name();
   }
 
+  @Override
+  public void authorize(AuthorizationService authorizationService, AuthenticationSubject subject)
+      throws UnauthorizedException {
+    authorizationService.authorizeSchemaWrite(
+        subject, type.keyspace(), null, Scope.CREATE, SourceAPI.GRAPHQL, ResourceKind.TYPE);
```

Comment:
Passing a `null` table name for all UDT permissions, similar to what is done in `StargateQueryHandler`.